src/Components/SearchBar/SearchBar.css

```
@@ -0,0 +1,22 @@
+@import url('https://fonts.googleapis.com/css2?family=Arimo&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Work+Sans:wght@500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=DM+Sans&display=swap');
+
+.Subtitle {
+    font-family: 'Work Sans', sans-serif;
```

when switching to the local version, you don't need the ",sans-serif;" part!
```suggestion
    font-family: 'Work Sans';
```